apps/desktop/src/components/exporting/utils/svg-generator.ts

```
@@ -301,36 +343,63 @@ export const generateDrillChartExportSVGs = async (args: {
-    const { canvas, canvasMarchersById } =
-        await initializeCanvasForRendering(args);
+    const { canvas, canvasMarchersById } = await initializeCanvasForRendering({
+        ...args,
+        propMarcherIds,
+    });
 
-    // If individual charts are enabled, initialize the output SVGs array for each marcher
+    // If individual charts are enabled, initialize the output SVGs array for each non-prop marcher
     // Otherwise, initialize the output SVGs for the main chart
-    if (individualCharts) marchers.forEach(() => outputSVGs.push([]));
+    if (individualCharts) nonPropMarchers.forEach(() => outputSVGs.push([]));
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)